javascript-light-bulb/main.js

```diff
@@ -0,0 +1,27 @@
+// create a JS storage for a reference to the HTML button.
+// create a JS storage for a reference to the HTML body background.
+
+// create a click event listener that will perform actions when the button is clicked.
+
+// define what will happen when the button is clicked:
+//   -if the button was previously set to dark, then
+//     -set the button to light mode
+//     -set the background to light mode
+//   -otherwise, if the button was previously set to light, then
+//     -set the button to dark mode
+//     -set the background to dark mode
+
+var $button = document.querySelector('button');
+var $background = document.querySelector('body');
+
+$button.addEventListener('click', handleClick);
+
+function handleClick(event) {
+  if ($button.className === 'bt-dark') {
```

Comment:
Instead of basing your functionality around the current classes on the DOM element, I would use a variable to keep track of if your lightbulb is "on" or "off". A boolean value would probably be a good choice. Remember what we discussed in Q&A, yesterday, we want all JavaScript functionality to not be reliant on data stored in the DOM as much as possible, you should have the information you need to do your work all centralized in your JavaScript.